### ISSUE TITLE
ovos-dinkum-listener times out before large-v2 fasterwhisper model downloads

### ISSUE THREAD

**Issue by mikejgray** [OpenVoiceOS/ovos-dinkum-listener#87]
I realize this is an edge case, but I set up local STT on my laptop running  OVOS and it took more than a minute to get the model downloaded, so the listener service failed repeatedly. This would only impact users installing from the manual installation script, but it might be nice to make it configurable for folks running local STT.

**Comment by JarbasAl** [comment]
is this an issue for the images/installer? this is systemd restarting the process right, not a bug in our code other than it being slow @mikejgray @builderjer @goldyfruit

**Comment by builderjer** [comment]
I would not say for images, Mike was using the manual_install.sh script which works similar but not near as good as ovos-installer.  If you are running an image it is set up one way without options.

**Comment by mikejgray** [comment]
> is this an issue for the images/installer? this is systemd restarting the process right, not a bug in our code other than it being slow @mikejgray @builderjer @goldyfruit 

It's not really an images/installer issue as this happens with faster-whisper, which doesn't come with those two things

**Comment by JarbasAl** [comment]
> > is this an issue for the images/installer? this is systemd restarting the process right, not a bug in our code other than it being slow @mikejgray @builderjer @goldyfruit
> 
> It's not really an images/installer issue as this happens with faster-whisper, which doesn't come with those two things

to rephrase, it is the systemd service created in all those circumstances that is rebooting the listener

images, installer script... if you are using the watchdog then systemd is the culprit here

**Comment by builderjer** [comment]
Exactly,  but to me, that is not a bug. It will keep trying to restart until it can. I think there is a setting for that in the service file, but if it times out there, it will have to be manually restarted. 

IMO, it should keep trying to restart until a model is loaded. Maybe a prompt that it is downloading a model and will be unresponsive for a while.

**Comment by builderjer** [comment]
The installer or any install script could ask if they want local stt and download the model then
